probe-dictionary/explore.js

```diff
@@ -431,6 +431,10 @@ function shortVersion(v) {
   return v.split(".")[0];
 }
 
+function getLatestVersion(channel) {
+  return Math.max.apply(null, Object.keys(gChannelInfo[channel].versions));
```

Comment:
The spread operator (`return Math.max(...gChannelInfo[channel].versions);`) has [excellent cross-browser support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Browser_compatibility) and is more concise.